doc/measuring-transaction-throughput.md

```diff
@@ -1,5 +1,70 @@
 # Running maximum transaction per second test
 
+Supercluster provides two missions that measure the maximum number of Transactions Per Second (TPS) stellar-core can support under some set of parameters:
+* `SimulatePubnetTier1Perf` measures maximum TPS using exclusively classic payments
+* `MaxTPSMixed` measures maximum TPS using a configurable mix of soroban invoke, soroban upload, and classic pay transactions.
+
+Other than the type of load generated, these two missions are identical. They both spin up a configurable network of stellar-core nodes, then measure the maximum TPS of the network using a binary search over a configurable range of TPS values. For each tested TPS value, the missions generate roughly 15 minutes of load, then check whether all nodes remained in sync for the duration of the test. If all nodes remained in sync, the missions consider the run successful and try again with a higher target TPS value. Otherwise, the missions consider the run a failure and retry with a lower target TPS value.
+
+The missions perform the binary search a configurable number of times, then average the highest successful values from each run. Upon completion, the missions emit a log line of the form, “Final tx rate averaged to 1000 over 3 runs for image ...".
+
+## Docker images with performance tests enabled
+
+To run these missions, you'll need a stellar-core docker image with performance tests enabled. The simplest way to get one is to use an image from the [Dockerhub stellar/unsafe-stellar-core repo](https://hub.docker.com/r/stellar/unsafe-stellar-core/tags) with `perftests` in the name. Note that `unsafe` in this case means "unsafe to use in production", as these are development builds that haven't necessarily undergone the same testing procedure as release builds.
```

Comment:
I added a warning to not run any `perftest` builds in production. I didn't want to just say "dangerous" without expanding on it because I don't want people to think that these images could harm the overall network (they can't). I tried to make it clear that these are dangerous in that they may corrupt your own local node state.